python/geom.py

```diff
@@ -434,9 +440,15 @@ def determinant(self):
         ])
         return sum1 - sum2
 
+    def conj(self):
+        return Matrix(self.c1.conj(), self.c2.conj(), self.c3.conj())
+
     def transpose(self):
         return Matrix(self.row(0), self.row(1), self.row(2))
 
+    def adjoint(self):
```

Comment:
In numpy this is `matrix.H`: https://docs.scipy.org/doc/numpy-1.13.0/reference/generated/numpy.matrix.H.html